libraries/botbuilder-dialogs-adaptive/src/luis/luisAdaptiveRecognizer.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * @module botbuilder-dialogs-adaptive
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { Recognizer } from '../recognizers';
+import { StringExpression, ArrayExpression } from 'adaptive-expressions';
+import { LuisPredictionOptions, LuisRecognizerOptionsV3, LuisRecognizer, LuisApplication } from 'botbuilder-ai';
+import { Configurable, DialogContext } from 'botbuilder-dialogs';
+import { BotTelemetryClient, Activity, ActivityTypes } from 'botbuilder-core';
+
+export class LuisAdaptiveRecognizer extends Configurable implements Recognizer {
+    /**
+     * Recognizers unique ID.
+     */
+    public id: string;
+
+    /**
+     * LUIS application ID.
+     */
+    public applicationId: StringExpression;
+
+    /**
+     * LUIS dynamic list.
+     */
+    public dynamicLists: ArrayExpression<any>;
+
+    /**
+     * LUIS endpoint to query.
+     * @remarks
+     * For example: "https://westus.api.cognitive.microsoft.com"
+     */
+    public endpoint: StringExpression;
+
+    /**
+     * Key used to talk to a LUIS endpoint.
+     */
+    public endpointKey: StringExpression;
+
+    /**
+     * External entity recognizer.
+     * @remarks
+     * This recognizer is run before calling LUIS and the results are passed to LUIS.
+     */
+    // public externalEntityRecognizer: Recognizer;
```

Comment:
Should this be uncommented?

Reply:
No... We can't support it right now as it requires changes to the base luis recognizer.